src/components/scenes/CryptoExchangeQuoteScene.tsx

```diff
@@ -3,12 +3,14 @@ import { EdgeAccount } from 'edge-core-js/types'
 import * as React from 'react'
 import { ScrollView, View } from 'react-native'
 import FastImage from 'react-native-fast-image'
+import { TouchableOpacity } from 'react-native-gesture-handler'
```

Comment:
This is the wrong `TouchableOpacity`. I think you want the one from React Native itself. Easy mistake to make with auto-import.

Reply:
ack. good catch. I've become too reliant on Skynet